geometric_features/aggregation/__init__.py

```diff
@@ -51,9 +51,12 @@ def get_aggregator_by_name(region_group):
                'Ocean Subbasins': {'prefix': 'oceanSubbasins',
                                    'date': '20201123',
                                    'function': subbasins},
-               'Greenland Regions': {'prefix': 'greenlandRegions',
+               'ISMIP6 Greenland Regions': {'prefix': 'ISMIP6GreenlandRegions',
```

Comment:
@xylar , do you prefer that the `prefix` for this and the NASA Greenland Regions to use lower/camel case for the acronyms like is done for the "ismip6Regions" prefix?

Reply:
It makes sense to me just to change the caps to make this consistent